scripts/jenkins/centos6/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM centos:6
+RUN yum groupinstall -y "Development Tools"
+RUN yum install -y rpmdevtools
+RUN yum install -y openssl yum-utils
+
+#
+#  Create build directory
+#
+RUN mkdir -p /usr/local/src/repositories
+WORKDIR /usr/local/src/repositories
+
+#
+#  Shallow clone the FreeRADIUS source
+#
+ARG source=https://github.com/FreeRADIUS/freeradius-server.git
+ARG release=v3.0.x
+
+RUN git clone --depth 1 --branch ${release} ${source}
+WORKDIR freeradius-server
+#
+#  Other requirements
+#
+
+# Use LTB's openldap packages intead of the distribution version to avoid linking against NSS
+RUN echo $'[ltb-project]\n\
```

Comment:
That file creation could be changed to provide that file into the filesystem and just use the command `COPY`

e.g:

```
COPY ltb-project.repo /etc/yum.repos.d/ltb-project.repo
```

ps: you should create that file into the same `Dockerfile` path.